cmd/fuzzbinder/README.md

```diff
@@ -1 +1,66 @@
 # Fuzzing Claims for revisions of a source code
+
+The goal of Fuzzbinder is to generate, sign and publish (in a transparency log) fuzzing claims for revisions of the source code of a given project.
+
+By following the steps described below, you will be able to generate fuzzing claims for your open-source project using the fuzzing reports generated by OSS-Fuzz and ClusterFuzz for your project, and the command line tool
+[`cmd/fuzzbinder`](/cmd/fuzzbinder/) that we have provided to generate fuzzing claims.
+
+## Generating fuzzing claims using the `cmd/fuzzbinder` tool
+
+If you have not already done so, consult the [`development guidelines`](./../../docs/development-guidelines.md) to find out what you need to install before following the steps below.
+
+### Step 1: Establish access to Google Cloud Storage
```

Comment:
I suggest you split this into 3 steps:

Step 1: Get access to ClusterFuzz crash reports and fuzzer statistics 
See https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary

Step 2 (optional): install google cloud sdk

Step 3: Authenticate to google cloud

Reply:
Thanks! I updated it. However, I don't see why step two is optional. Unless the user does not want to test that they have access to the buckets before generating the claims they need gsutil, right?

Reply:
Yes. It is optional if it is installed from before.